src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.EasyRequest.cs

```diff
@@ -96,6 +96,34 @@ internal void InitializeCurl()
 
             public void EnsureResponseMessagePublished()
             {
+                // If this is the response hasn't been published yet, do any finaly processing of the response 
+                // message before it's published.
```

Comment:
"If this is the response hasn't been published yet" -> "If this response hasn't been published yet"